modules/simple_forms_api/app/models/simple_forms_api/vba_21_4140.rb

```diff
@@ -30,10 +37,27 @@ def middle_initial
       data.dig('full_name', 'middle')&.[](0)
     end
 
+    def phone_alternate
+      data['mobile_phone'].insert(-8, '-').insert(-5, '-')
+    end
+
+    def phone_primary
+      data['home_phone'].insert(-8, '-').insert(-5, '-')
```

Comment:
This formatting could be abstracted, and probably will when we move on to other forms, but for now, it was simpler and more efficient to repeat it.

Reply:
Thanks for the comment here.